### PR TITLE
add converter and kienzler_stuttgart static data

### PR DESCRIPTION
**Pull request by ThorstenFroehlinghaus**

### CODE REVIEW

File: csv2geojson.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import sys
+import csv
+import json
+
+argv = sys.argv[1:]
+if len(argv) == 0:
+    print("no csv file specified")
+    sys.exit(2)
+csvfile = argv[0]
```
Comment (the-infinity) [inline csv2geojson.py:9]: Naming: this is not a file, but a path.

Better approach of this: using Pathlib. Then it's `csvfile = Path(argv[0])`, and `with csvfile.open(...) as ...`.

File: csv2geojson.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                properties[column] = int(row[column])
+        if 'park_and_ride_type' in row and row['park_and_ride_type'] != '':
+            properties['park_and_ride_type'] = [row['park_and_ride_type']]
+        if 'DHID' in row and row['DHID'] != '':
+            properties['external_identifiers'] = { 'type': 'DHID', 'value': row['DHID'] }
+        features.append({ 'type': 'Feature', 'properties': properties, 'geometry': { 'type': 'Point', 'coordinates': [float(row['lon']), float(row['lat'])] } })
+
+jsonfile = csvfile.replace('.csv', '.geojson')
+jsondata = { 'type': 'FeatureCollection', 'features': features }
+with open(jsonfile, 'w', encoding='utf-8') as file:
```
Comment (the-infinity) [inline csv2geojson.py:36]: UTF8 is default for quite some time, so you don't need that.

File: csv2geojson.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import json
+
+argv = sys.argv[1:]
+if len(argv) == 0:
+    print("no csv file specified")
+    sys.exit(2)
+csvfile = argv[0]
+
+features = []
+with open(csvfile, 'r', encoding='utf-8') as file:
```
Comment (the-infinity) [inline csv2geojson.py:12]: UTF8 is default for quite some time, so you don't need that.

File: csv2geojson.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            print(f'Attributes \'lat\'/\'lon\' missing in row {row}')
+            continue
+        properties = { 'uid': row['uid'] }
+        for column in ['address', 'type']:
+            if column in row and row[column] != '':
+                properties[column] = row[column]
+        for column in ['max_height', 'max_width', 'max_depth']:
+            if column in row and row[column] != '' and row[column].isdigit():
+                properties[column] = int(row[column])
+        if 'park_and_ride_type' in row and row['park_and_ride_type'] != '':
```
Comment (the-infinity) [inline csv2geojson.py:28]: You can just write `if not park_and_ride_type.get('park_and_ride_type')` to simplify this (same at other places)

File: csv2geojson.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import sys
```
Comment (the-infinity) [inline csv2geojson.py:1]: Would be good to have some documentation in the README for this script

File: csv2geojson.py [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+        for column in ['address', 'type']:
+            if column in row and row[column] != '':
+                properties[column] = row[column]
+        for column in ['max_height', 'max_width', 'max_depth']:
+            if column in row and row[column] != '' and row[column].isdigit():
+                properties[column] = int(row[column])
+        if 'park_and_ride_type' in row and row['park_and_ride_type'] != '':
+            properties['park_and_ride_type'] = [row['park_and_ride_type']]
+        if 'DHID' in row and row['DHID'] != '':
+            properties['external_identifiers'] = { 'type': 'DHID', 'value': row['DHID'] }
```
Comment (the-infinity) [inline csv2geojson.py:31]: External identifiers should be a list of dicts, not just a dict. Like this: `[{'type': 'DHID', 'value': 123}, {'type': 'OSM', 'value': 321}]`.

File: csv2geojson.py [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+        features.append({ 'type': 'Feature', 'properties': properties, 'geometry': { 'type': 'Point', 'coordinates': [float(row['lon']), float(row['lat'])] } })
+
+jsonfile = csvfile.replace('.csv', '.geojson')
+jsondata = { 'type': 'FeatureCollection', 'features': features }
+with open(jsonfile, 'w', encoding='utf-8') as file:
+    json.dump(jsondata, file, ensure_ascii=False, indent=4)
```
Comment (the-infinity) [inline csv2geojson.py:37]: Please add empty lines at the end of each file, then these forbidden signs will disappear.

**Comment by ThorstenFroehlinghaus** [comment]
I updates csv2geojson.py and kienzler_stuttgart.geojson

**Comment by ThorstenFroehlinghaus** [comment]
added new kienzler sources

File: README.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+## csv2geojson.py
+
+This script converts a csv file into a geojson file, e.g.:
+```bash
+py csv2geojson.py kienzler_stuttgart.csv
```
Comment (the-infinity) [inline README.md:5]: This won't work in bash :) Please change it to `python`.

**Review by the-infinity — APPROVED** [review]

**Pull request merged**